scripts/update-cloudfront-function.ts

```diff
@@ -0,0 +1,55 @@
+import cli from 'cli'
+
+import {
+    createDefaultStorageProvider,
+    getStorageProvider
+} from '@etherealengine/server-core/src/media/storageprovider/storageprovider'
+import {createFeathersKoaApp} from "@etherealengine/server-core/src/createApp";
+import {ServerMode} from "@etherealengine/server-core/src/ServerState";
+import {InstalledRoutesInterface} from "@etherealengine/common/src/interfaces/Route";
+import {Application} from "@feathersjs/feathers";
+
+cli.enable('status')
+
+const PAGE_SIZE = 1
+
+
+const getAllRoutes = async (app: Application, routes: InstalledRoutesInterface[], skip: number) => {
+    const routePage = await app.service('route').find({
```

Comment:
Instead of 'route' we can use `routePath` variable that is exported from route.scheme.ts

> We are trying to use single point of reference variable for service name, db name, etc.